modules/errorhandler-core/ErrorHandler.asm

```diff
@@ -478,12 +478,9 @@ ErrorHandler_SetupVDP:	__global
 	lea 	VDP_Ctrl, a5 			; a5 = VDP_Ctrl
 	lea 	-4(a5), a6				; a6 = VDP_Data
 
-	; Make sure there are no pending writes to VDP
-	tst.w	(a5)
-
-	; Make sure there are no DMA's occuring, otherwise wait
+	; Make sure there are no DMA's occurring, otherwise wait
 	@wait_dma:
-		move.w	(a5), ccr				; is DMA occuring?
+		move.w	(a5), ccr				; is DMA occurring? (also clears VDP write flag)
```

Comment:
That's a nice little optimization. I've been thinking of this one, but then forgot :)

Reply:
Why use a garbage read to clear the write flag when you can do something productive with it? I first saw this in some old init code of MarkeyJester's, and it's stuck with me ever since.